components/Layout.js

```diff
@@ -1,38 +1,26 @@
-'use strict';
-
 const React = require('react');
-const { KeyboardAvoidingView, SafeAreaView, ScrollView, Keyboard } = require('react-native');
-const { default: Styled } = require('styled-components/native');
-
-const Header = Styled.View`
-    background: ${({ theme }) => theme.palette.BRAND};
-`;
+const {
+    KeyboardAvoidingView,
+    ScrollView,
+    Keyboard,
+    Platform
+} = require('react-native');
 
-const HeaderText = Styled.Text`
-    padding: ${({ theme }) => theme.spacing(0.25)}px 0 ${({ theme }) => theme.spacing(1.5)}px;
-    font-size: ${({ theme }) => theme.spacing(1.125)}px;
-    font-weight: bold;
-    text-align: center;
-    color: ${({ theme }) => theme.palette.ETCH};
-`;
+module.exports = function Layout(Component) {
```

Comment:
should we consider renaming `Layout` to clarify that it's an HOC now e.g. `withLayout`?